src/main.ts

```diff
@@ -45,6 +45,41 @@ export default class VimReadingViewNavigation extends Plugin {
 			}
 			return true;
 		});
+        this.navScope.register([], 'g', (evt: KeyboardEvent) => {
+            const leaf = app.workspace.getActiveViewOfType(MarkdownView);
+			if (
+				leaf.getMode() === 'preview' &&
+				this.displayValue(leaf) === 'none' &&
+                evt.key === 'g'
+			) {
+				this.scrollTop(leaf);
+				return false;
+			}
+			return true;
+		});
+		this.navScope.register([], 'g', (evt: KeyboardEvent) => {
+			const leaf = app.workspace.getActiveViewOfType(MarkdownView);
+			if (
+				leaf.getMode() === 'preview' &&
+				this.displayValue(leaf) === 'none' &&
+                evt.key === 'G'
```

Comment:
Why are you registering lowercase g twice and checking once the uppercase variant? 
Could that also occur in addition to the lowercase variant and shiftKey set to true?

Reply:
The first on is for when the user presses `g`, the second is for when the user presses `g` AND caps lock is on, and the last is for when the user presses `Shift+g`.

Reply:
Theres probably a better way to do this. Ill read the standard browser APIs that you sent and see if I can come up with something better.

Reply:
Ah, I didn’t know that caps lock does that. Can you add a short comment for it?

Reply:
Yep of course!